src/booking/OfferRequests/OfferRequestsTypes.ts

```diff
@@ -316,12 +318,12 @@ export interface CreateOfferRequestSlice {
   /**
    * The inclusive time range for the arrival of the slice
    */
-  arrival_time?: string | null
+  arrival_time?: TimeRangeFilter | null
```

Comment:
Can we remove the undefined please?

Reply:
The concept of null or undefined to the API is the same I guess. Whether we send the attribute set to null or don't send it at all. But i did notice that when we do redefine or reference some of these types on the dashboard they may mismatch because one side expects undefined to be part of the type vs not. Or some places where we only check if `null`. Not big deal though if you'd prefer to keep the undefined so we don't need to add to objects when using them.